### ISSUE TITLE
Crucible website is down

### ISSUE THREAD

**Issue by Scottwint** [vgulerianb/crucible#1]
Every YouTube video link I pasted on crucible always comes back with the response, "unable to process given video, try another one." What's wrong with the site? Kindly look into it

**Comment by vgulerianb** [comment]
I have hit the hard limit of allowed OpenAI usage for this month, this should be fixed once its increased

**Comment by Scottwint** [comment]
Thank you for your response. Does the usage limit refresh every month?

**Comment by vgulerianb** [comment]
Yes, but I have raised a request to increase my allowed limit, if approved things should start working again in a couple of days. Additionally you can clone this repo and run crucible locally with your own API keys

**Comment by naveenfaclon** [comment]
Website is down, not able to open the application.
